django-stubs/core/exceptions.pyi

```diff
@@ -24,15 +24,20 @@ class MiddlewareNotUsed(Exception): ...
 class ImproperlyConfigured(Exception): ...
 class FieldError(Exception): ...
 
-NON_FIELD_ERRORS: str
+NON_FIELD_ERRORS: Literal["__all__"]
 
 class ValidationError(Exception):
-    error_dict: Any = ...
-    error_list: Any = ...
-    message: Any = ...
-    code: Any = ...
-    params: Any = ...
-    def __init__(self, message: Any, code: Optional[str] = ..., params: Optional[Mapping[str, Any]] = ...) -> None: ...
+    error_dict: Dict[str, ValidationError] = ...
+    error_list: List[ValidationError] = ...
+    message: str = ...
+    code: Optional[str] = ...
+    params: Optional[Dict[str, Any]] = ...
+    def __init__(
+        self,
+        message: Union[str, ValidationError, Dict[str, Any], List[Any]],
```

Comment:
Can be an arbitrarily nested combination of strings, `ValidationError`s, and dicts/lists containing more dict/slists/str/ValidationError.

Can this be expressed in the type system currently?

Reply:
Not really 😞 
`Any` is good enough for now.